ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.template.html

```diff
@@ -451,16 +450,7 @@ <h4 ng-if="model.hasEnrichers() && filteredEnrichers.length === 0">No enrichers
 
         <script type="text/ng-template" id="blueprint-composer/component/spec-editor/section-others.html" defer-to-preexisting-id="true"></script>
     </div>
-    <div class="toolbar" ng-if="specEditor.needToolbar">
-        <div class="list-group">
-            <a href="" class="list-group-item ng-scope" title="{{section.type.displayName}} configuration"
-               ng-repeat="section in specEditor.sections track by $index"
-               ng-class="{'active': $root.selectedSection === section}"
-               ng-click="$root.selectedSection = section">
-                <i class="fa fa-fw" ng-class="section.icon"></i>
-            </a>
-        </div>
-    </div>
+    <div class="toolbar" ng-if="specEditor.needToolbar"/>
```

Comment:
Is-it possible to remove it or is-it referenced somewhere ?
Maybe the right tool bar is added inside it ?

Reply:
yes it is extactly it :)